cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -144,10 +144,15 @@ public AbstractConnectOperator(Vertx vertx, PlatformFeaturesAvailability pfa, St
         this.operatorNamespaceLabels = config.getOperatorNamespaceLabels();
         this.pfa = pfa;
         this.port = port;
-
         // Setup metrics for connectors
-        Tags metricTags = Tags.of(Tag.of("kind", KafkaConnector.RESOURCE_KIND));
 
+        Tags metricTags;
+
+        if (config.getCustomResourceSelector() != null) {
+            metricTags = Tags.of(Tag.of("kind", KafkaConnector.RESOURCE_KIND), Tag.of("selector", config.getCustomResourceSelector().toSelectorString()));
+        } else {
+            metricTags = Tags.of(Tag.of("kind", KafkaConnector.RESOURCE_KIND));
+        }
```

Comment:
Does the Connector operator really follow the selector labels? Doesn't it follow only the connect labels? If that is the case, I'm not sure this should apply to the metric.

Reply:
I ran the tests